Makefile.am

```diff
@@ -23,14 +24,15 @@ ACLOCAL_AMFLAGS = -I aclocal
 
 install-data-hook:
 	if [ ! -d $(DESTDIR)$(statedir) ]; then mkdir -p $(DESTDIR)$(statedir); fi
-	touch $(DESTDIR)$(statedir)/xtab; chmod 644 $(DESTDIR)$(statedir)/xtab
+	touch $(DESTDIR)$(statedir)/etab; chmod 644 $(DESTDIR)$(statedir)/etab
```

Comment:
etab and rmtab shouldn't appear here, right?

Reply:
I am looking at upstream's version and they kept etab references which probably threw me off: https://git.launchpad.net/ubuntu/+source/nfs-utils/tree/Makefile.am?h=applied/ubuntu/noble-devel

I'll restore https://github.com/delphix/nfs-utils/commit/fb92473ed9583e4af5f25a3f36c18053e026c077 as-is but shouldn't rmtab entries remain? Upstream still has them